helm/dagster/values.yaml

```diff
@@ -999,6 +999,10 @@ dagsterDaemon:
     # run launcher supports resuming runs, otherwise defaults to 0.
     maxResumeRunAttempts: ~
 
+  runRetries:
```

Comment:
will link to docs once https://app.graphite.dev/github/pr/dagster-io/dagster/8367 lands